src/jumbo.c

```diff
@@ -300,6 +300,19 @@ char *strlwr(char *s)
 }
 #endif
 
+void memcpylwr(char *dest, const char *src, size_t n)
```

Comment:
Would we get benefit from this function ?

```c
int strnzcpylwr(char *des, const char *src, size_t n) {  }
```
to do the work of strnzcpy with lc at same time ?

Reply:
Thanks for the idea, based on this and the ASAN warnings above (as pointed out by @claudioandre), I added ac3329626, let's see how it works out.